lib/util.js

```diff
@@ -0,0 +1,23 @@
+function getParameterByName(name) {
```

Comment:
This seems error prone. I'd prefer to just use something like [parseUri](http://blog.stevenlevithan.com/archives/parseuri).